<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Detail</class>
 <widget class="QMainWindow" name="Detail">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>642</width>
    <height>192</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>642</width>
    <height>192</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Detail Customer</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="font">
           <font>
            <family>Arial Black</family>
            <pointsize>14</pointsize>
            <weight>10</weight>
            <italic>false</italic>
            <bold>false</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 87 14pt &quot;Arial Black&quot;;</string>
          </property>
          <property name="text">
           <string>Prénom :</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="font">
           <font>
            <family>Arial Black</family>
            <pointsize>14</pointsize>
            <weight>10</weight>
            <italic>false</italic>
            <bold>false</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 87 14pt &quot;Arial Black&quot;;</string>
          </property>
          <property name="text">
           <string>Nom :</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_4">
          <property name="font">
           <font>
            <family>Arial Black</family>
            <pointsize>14</pointsize>
            <weight>10</weight>
            <italic>false</italic>
            <bold>false</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 87 14pt &quot;Arial Black&quot;;</string>
          </property>
          <property name="text">
           <string>Adresse :</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="lblPrenom">
          <property name="font">
           <font>
            <family>Arial</family>
            <pointsize>14</pointsize>
            <weight>10</weight>
            <italic>false</italic>
            <bold>false</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color : rgba(0,0,0,0);
border: 0px;
font: 87 14pt &quot;Arial&quot;;</string>
          </property>
          <property name="text">
           <string>JOE</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lblNom">
          <property name="font">
           <font>
            <family>Arial</family>
            <pointsize>14</pointsize>
            <weight>10</weight>
            <italic>false</italic>
            <bold>false</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color : rgba(0,0,0,0);
border: 0px;
font: 87 14pt &quot;Arial&quot;;</string>
          </property>
          <property name="text">
           <string>DALTON</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lblAdresse">
          <property name="font">
           <font>
            <family>Arial</family>
            <pointsize>14</pointsize>
            <weight>10</weight>
            <italic>false</italic>
            <bold>false</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color : rgba(0,0,0,0);
border: 0px;
font: 87 14pt &quot;Arial&quot;;</string>
          </property>
          <property name="text">
           <string>Avenue des pins 14</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="label_8">
          <property name="font">
           <font>
            <family>Arial Black</family>
            <pointsize>14</pointsize>
            <weight>10</weight>
            <italic>false</italic>
            <bold>false</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 87 14pt &quot;Arial Black&quot;;</string>
          </property>
          <property name="text">
           <string>NPA/Localité :</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_9">
          <property name="font">
           <font>
            <family>Arial Black</family>
            <pointsize>14</pointsize>
            <weight>10</weight>
            <italic>false</italic>
            <bold>false</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 87 14pt &quot;Arial Black&quot;;</string>
          </property>
          <property name="text">
           <string>Mobile :</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_10">
          <property name="font">
           <font>
            <family>Arial Black</family>
            <pointsize>14</pointsize>
            <weight>10</weight>
            <italic>false</italic>
            <bold>false</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 87 14pt &quot;Arial Black&quot;;</string>
          </property>
          <property name="text">
           <string>Téléphone :</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_14">
          <property name="font">
           <font>
            <family>Arial Black</family>
            <pointsize>14</pointsize>
            <weight>10</weight>
            <italic>false</italic>
            <bold>false</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 87 14pt &quot;Arial Black&quot;;</string>
          </property>
          <property name="text">
           <string>Email :</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QLabel" name="lblNPA">
          <property name="font">
           <font>
            <family>Arial</family>
            <pointsize>14</pointsize>
            <weight>10</weight>
            <italic>false</italic>
            <bold>false</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color : rgba(0,0,0,0);
border: 0px;
font: 87 14pt &quot;Arial&quot;;</string>
          </property>
          <property name="text">
           <string>1570</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lblNumero">
          <property name="font">
           <font>
            <family>Arial</family>
            <pointsize>14</pointsize>
            <weight>10</weight>
            <italic>false</italic>
            <bold>false</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color : rgba(0,0,0,0);
border: 0px;
font: 87 14pt &quot;Arial&quot;;</string>
          </property>
          <property name="text">
           <string>078 000 00 00</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lblTelephone">
          <property name="font">
           <font>
            <family>Arial</family>
            <pointsize>14</pointsize>
            <weight>10</weight>
            <italic>false</italic>
            <bold>false</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color : rgba(0,0,0,0);
border: 0px;
font: 87 14pt &quot;Arial&quot;;</string>
          </property>
          <property name="text">
           <string>021 000 00 00</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lblEmail">
          <property name="font">
           <font>
            <family>Arial</family>
            <pointsize>14</pointsize>
            <weight>10</weight>
            <italic>false</italic>
            <bold>false</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color : rgba(0,0,0,0);
border: 0px;
font: 87 14pt &quot;Arial&quot;;</string>
          </property>
          <property name="text">
           <string>joe.dalton@jimail.com</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="btnContracts">
        <property name="maximumSize">
         <size>
          <width>150</width>
          <height>50</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>12</pointsize>
          <weight>10</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 87 12pt &quot;Arial&quot;;</string>
        </property>
        <property name="text">
         <string>Liste des contrats</string>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_5">
        <property name="styleSheet">
         <string notr="true">font: 87 14pt &quot;Arial Black&quot;;</string>
        </property>
        <property name="text">
         <string>Nb Contrats:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lbl_nbContracts">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
